README.md

```diff
@@ -118,12 +118,12 @@ public class DaggerScreenScooper extends ScreenScooper {
         DaggerInjector parentDagger = DaggerInjector.fromScoop(parentScoop);
 
         // each controller have an attribute that points to dagger module
-        ControllerModule controllerModule = screen.getController().getAnnotation(ControllerModule.class);
+        DaggerModule daggerModule = screen.getController().getAnnotation(DaggerModule.class);
 
         DaggerInjector screenInjector;
 
         try {
-            Object module = controllerModule.value().newInstance();
+            Object module = daggerModule.value().newInstance();
```

Comment:
@itspbj look below at exception you generate. you are saying module not specified for controller. It's not longer true.